pandas/core/config_init.py

```diff
@@ -539,6 +539,26 @@ def use_inf_as_na_cb(key) -> None:
     )
 
 
+# TODO better name?
+copy_on_write_doc = """
+: bool
+    Use new copy-view behaviour using Copy-on-Write. Defaults to False,
+    unless overridden by the 'PANDAS_COPY_ON_WRITE' environment variable (needs
```

Comment:
Maybe good to clarify that the environment variable must be set to `"1"` for true as above it states this should be `bool` as well